docs/guides/bnn-architecture.md

```diff
@@ -19,8 +19,8 @@ import larq as lq
 ...
 x_out = lq.layers.QuantDense(
     units=1024,
-    input_quantizer=lq.quantizers.ste_sign,
-    kernel_quantizer=lq.quantizers.ste_sign,
+    input_quantizer=lq.quantizers.SteSign(clip_value=1.0),
+    kernel_quantizer=lq.quantizers.SteSign(clip_value=1.0),
     kernel_constraint=lq.constraints.WeightClip(clip_value=1.0),
```

Comment:
It's nice that the `SteSign()` instantiations are now more similar to the `WeightClip()` ones.